src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/SemanticRange.cs

```diff
@@ -26,8 +28,11 @@ public SemanticRange(int kind, Range range, int modifier, bool fromRazor)
     /// If we produce a token, and a delegated server produces a token, we want to prefer ours, so we use this flag to help our
     /// sort algorithm, that way we can avoid the perf hit of actually finding duplicates, and just take the first instance that
     /// covers a range.
+    /// Presently used only in Debug
```

Comment:
As per previous comments, my preference would be to remove this altogether, including the constructor parameter. If you disagree, and its going to stay, then consider moving this XML comment inside the `#if DEBUG`. As it stands, in release mode this will be seen as documentation about the `CompareTo` method, and whilst nobody will ever read said documentation, it would still be surprising.